common/menu.c

```diff
@@ -439,15 +476,26 @@ void menuLoop() {
 
             drawEntry(me, entry_start_x + menu->xPos, is_active);
         }
+        
+        int getX = GetTextXCoordinate(interuiregular18, 1180, textGetString(StrId_ThemeMenu), 'r');
 
+        if(hbmenu_state == HBMENU_THEME_MENU) {
+            DrawText(interuiregular18, getX, 0 + 47, themeCurrent.textColor, textGetString(StrId_ThemeMenu));
+        } else {
+            DrawText(interuiregular18, getX, 0 + 47, themeCurrent.textColor, textGetString(StrId_ThemeMenu));
```

Comment:
Comment out these two lines.

Reply:
you dont like it? :/

Reply:
Yeah just disable for now, can figure out what to do with it later.

Reply:
i cant find the plus button for the light theme.... weird

Reply:
yes

Reply:
ok, i am commenting them out and about to commit

Reply:
is there a file that shows all of the characters and their unicode numbers? i might try creating one of those....

Reply:
You have to load the shared-font with some font tool.